recipe/build.sh

```diff
@@ -11,6 +11,13 @@ if [[ "$CONDA_BUILD_CROSS_COMPILATION" == "1" ]]; then
     CMAKE_ARGS="$CMAKE_ARGS -DLLVM_CONFIG_PATH=$BUILD_PREFIX/bin/llvm-config -DMLIR_TABLEGEN_EXE=$BUILD_PREFIX/bin/mlir-tblgen"
 fi
 
+if [[ "$variant" == "emscripten"* ]]; then
+    CMAKE_ARGS="$CMAKE_ARGS \
+        -DLLVM_DEFAULT_TARGET_TRIPLE=wasm32-unknown-emscripten \
+        -DLLVM_TARGETS_TO_BUILD=WebAssembly \
+        -DLLVM_ENABLE_PROJECTS='clang;flang;mlir'"
```

Comment:
I think we need to remove this; this is rebuilding way too many things that are already there and will blow through the 6h time budget that we have (and shouldn't be necessary 🤞)
```suggestion
```